libbeat/outputs/elasticsearch/client.go

```diff
@@ -551,17 +551,18 @@ func (client *Client) PublishEvent(data outputs.Data) error {
 func (client *Client) LoadTemplate(templateName string, template map[string]interface{}) error {
 
 	path := "/_template/" + templateName
-	status, _, err := client.request("PUT", path, "", nil, template)
+	return client.LoadJson(path, template)
+}
 
+func (client *Client) LoadJson(path string, json map[string]interface{}) error {
+	status, _, err := client.request("PUT", path, "", nil, json)
 	if err != nil {
-		return fmt.Errorf("Template could not be loaded. Error: %s", err)
+		return fmt.Errorf("couldn't load json. Error: %s", err)
 	}
-	if status != 200 {
-		return fmt.Errorf("Template could not be loaded. Status: %v", status)
+	if status > 300 {
```

Comment:
That means all 200 codes are fine? Why was this change needed? Do we get other codes for loading the kibana files?